### DISCUSSION TITLE
After a updating, the opening time surges.

### DISCUSSION THREAD

**Discussion by thudjx** [LazyVim/LazyVim#5192] [Q&A]
Hello everyone, 
I have just updated neovim to 0.10.2 and and lazyvim to the newest 14.5.0. But compared to my good old version (cannot remember exactly what it is), now the openning takes about 2.0 s, which is totally unbearable!

I don't want to roll back, and I am wondering are there any methods to monitor the performance problem to improve on it? In fact, I don't have to use most the fancy plugins; I am just some lazy person wanting some plug and play distribution of neovim that is light.

**Comment by folke** [comment]
That's not normal of course.
Startup time has actually improved if anything.
